cmd/set/set.go

```diff
@@ -0,0 +1,68 @@
+package set
+
+// maybe call this parameter "activate"?
+import (
+	"errors"
+	glooconnectcmd "github.com/solo-io/gloo-connect/pkg/cmd"
+	"github.com/solo-io/gloo-connect/pkg/runner"
+	"github.com/solo-io/gloo/pkg/bootstrap/configstorage"
+	"github.com/spf13/cobra"
+)
+
+type serviceFlagsType struct {
+	retries uint32
+	http    bool
+}
+
+var serviceFlags = serviceFlagsType{}
+
+func Cmd(rc *runner.RunConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set",
+		Short: "set service",
+	}
+	cmd.AddCommand(cmdSetServices(rc))
+	return cmd
+}
+
+func cmdSetServices(rc *runner.RunConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "service",
+		Short: "set service",
+		RunE: func(c *cobra.Command, args []string) error {
+			err := validateServiceParams(serviceFlags, args)
+			if err != nil {
+				return err
+			}
+			store, err := configstorage.Bootstrap(rc.Options)
+			if err != nil {
+				return err
+			}
+
+			gc := glooconnectcmd.GlooClient{Store: store}
+
+			return gc.ConfigureService(args[0], serviceFlags.retries)
+		},
+	}
+	cmd.PersistentFlags().Uint32VarP(&serviceFlags.retries, "retries", "", 0, "number of retries")
```

Comment:
I would mention that this is only valid in http mode in the help text

Reply:
good point, took another pass at it. Might want to revisit